libc/include/sys/ioctl.yaml

```diff
@@ -5,4 +5,10 @@ macros: []
 types: []
 enums: []
 objects: []
-functions: []
+functions:
```

Comment:
We should fix the "standards" list here while we're making this header usable.  POSIX definitely does not specify `ioctl` or `<sys/ioctl.h>`.  Both `bsd` and `linux` are reasonable things to list here.